core/src/dird/ua_select.cc

```diff
@@ -1124,6 +1174,9 @@ int DoPrompt(UaContext* ua,
   PoolMem pmsg(PM_MESSAGE);
   BareosSocket* user = ua->UA_sock;
 
+  int windowsize = 80;
+  int min_lines_threshold = 20;
```

Comment:
```suggestion
  constexpr int windowsize = 80;
  constexpr int min_lines_threshold = 20;
```
you can make these `constexpr` so they'll end up as a compile-time constant instead of an actual variable.